client/my-sites/site-settings/settings-newsletter/main.tsx

```diff
@@ -0,0 +1,114 @@
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import DocumentHead from 'calypso/components/data/document-head';
+import FormattedHeader from 'calypso/components/formatted-header';
+import Main from 'calypso/components/main';
+import isSiteAutomatedTransfer from 'calypso/state/selectors/is-site-automated-transfer';
+import { getSiteUrl, isJetpackSite } from 'calypso/state/sites/selectors';
+import { IAppState } from 'calypso/state/types';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { NewsletterSettingsSection } from '../reading-newsletter-settings';
+import wrapSettingsForm from '../wrap-settings-form';
+
+export type SubscriptionOptions = {
+	invitation: string;
+	comment_follow: string;
+};
+
+type Fields = {
+	subscription_options?: SubscriptionOptions;
+	wpcom_featured_image_in_email?: boolean;
+	wpcom_subscription_emails_use_excerpt?: boolean;
+	sm_enabled?: boolean;
+};
+
+const getFormSettings = ( settings: unknown & Fields ) => {
+	if ( ! settings ) {
+		return {};
+	}
+
+	const {
+		subscription_options,
+		wpcom_featured_image_in_email,
+		wpcom_subscription_emails_use_excerpt,
+		sm_enabled,
+	} = settings;
+
+	return {
+		...( subscription_options && { subscription_options } ),
+		wpcom_featured_image_in_email: !! wpcom_featured_image_in_email,
+		wpcom_subscription_emails_use_excerpt: !! wpcom_subscription_emails_use_excerpt,
+		sm_enabled: !! sm_enabled,
+	};
+};
+
+const connectComponent = connect( ( state: IAppState ) => {
```

Comment:
I may be wrong, but it looks to me as if we are not using any of the connected props. 🤔

Reply:
Good catch! This also came from Reading settings I think — a good for a clean-up afterwards, or in this PR.